docs/guides/api.en.md

```diff
@@ -345,7 +471,344 @@
 
 ---
 
-## Patching : Stage 1 (Loading)
+## Settings
+
+??? example "AddMenu `method`"
+    <h3>Summary</h3>
+    Adds an entry to the in-game mod menu, which lets users configure settings at runtime.
+    <h3>Example</h3>
+    ```c#
+    // Example : Adds a slider to the mod menu to configure the amount of tiles walked by turn.
+    // The slider will have a range from 2 to 5, with a default value of 4.
+    Msl.AddMenu("MyMod", new UIComponent(name: "Tiles walked by turn", associatedGlobal: "tiles_by_turn", UIComponentType.Slider, (2, 5), 4)
+    ```
+    <h3>Arguments</h3>
+
+    | **Type** | **Name** | **Description**|
+    | :---: | :---: | :--- |
+    | `String` | `name` | The name of the category in which to place the new entry. |
+    | `UIComponent` | `components` | The component(s) to add to the mod menu. Can be a `CheckBox`, `ComboBox` or `Slider`. |
+
+    <h3>Returns</h3>
+
+    | **Type** | **Description** |
+    | :--- | :--- |
+    | `N/A` | N/A |
+
+??? example "AddCreditDisclaimerRoom `method`"
+    <h3>Summary</h3>
+    Adds your mod's name and authors to MSL's credits room, which is shown before the main menu.
+    <h3>Example</h3>
+    ```c#
+    AddCreditDisclaimerRoom("MyMod", "Myself", "Someone else", "Another person", "etc...")
+    ```
+    <h3>Arguments</h3>
+
+    | **Type** | **Name** | **Description**|
+    | :---: | :---: | :--- |
+    | `String` | `modName` | The name under which to show your mod in the credits room. |
+    | `String[]` | `authors` | The authors of the mod. |
+
+    <h3>Returns</h3>
+
+    | **Type** | **Description** |
+    | :--- | :--- |
+    | `N/A` | N/A |
+
+??? example "AddCustomDisclaimerRoom `method`"
+    <h3>Summary</h3>
+    Adds a custom disclaimer room to the game, which is shown after the credit room but before the main menu.
+    <h3>Example</h3>
+    ```c#
+    Msl.AddCustomDisclaimer("r_myDisclaimerRoom", myOverlay)
+    ```
+    <h3>Arguments</h3>
+
+    | **Type** | **Name** | **Description**|
+    | :---: | :---: | :--- |
+    | `String` | `roomName` | The name of the room to create. |
+    | `UndertaleRoom.GameObject` | `overlay` | The gameobject to show in the room. This should act as an overlay and contain all you want to display. |
+
+    <h3>Returns</h3>
+
+    | **Type** | **Description** |
+    | :--- | :--- |
+    | `N/A` | N/A |
+
+---
+
+## Loot
+
+??? example "AddLootTable `method`"
+    <h3>Summary</h3>
+    Creates a new loot table, which is a weighted list of gameobjects that can be found/dropped.
+    <h3>Example</h3>
+    ```c#
+    // Creates a new loot table with guaranteed items and items that have various chances of dropping.
+    // Also handles empty drops and items with rarity / durability.
```

Comment:
for later add what durability and rarity are and what can of values is expected in listItems

Reply:
Leaving it up to you, you know best for that method.